packages/main/src/Slider.hbs

```diff
@@ -1,7 +1,8 @@
 {{>include "./SliderBase.hbs"}}
 
 {{#*inline "handles"}}
-	<div class="ui5-slider-handle" style="{{styles.handle}}">
+	<div class="ui5-slider-handle" style="{{styles.handle}}" aria-labelledby="{{_id}}-handle-label">
+	<span id="{{_id}}-handle-label" aria-hidden="true" class="ui5-hidden-text">Slider Handle</span>

```

Comment:
Should not we make sure that the "Slider Handle" text is translated.

Reply:
I'll revert this attribute addition. We decided to implement accessibility features on a later stage, after discussions and planning by the acc. experts.